geopandas/tools/_show_versions.py

```diff
@@ -64,6 +70,15 @@ def _get_C_info():
     except Exception:
         gdal_dir = None
 
+    if gdal_version is None:
+        try:
+            import pyogrio
+
+            gdal_version = pyogrio.__gdal_version_string__
+            gdal_dir = None
```

Comment:
@brendan-ward we currently have some internal cython functionality to set the GDAL data directory, but we could maybe also expose a function to get the path?

Reply:
PROJ has APIs to get this (and pyproj exposes that as `pyproj.datadir.get_data_dir()`. GDAL doesn't seem to have equivalents, but we could also check which directory is being used in practice, in a similar way how we check that the data is being found:

```
def get_gdal_data_path():
    cdef const char *path_c = CPLFindFile("gdal", "header.dxf")
    if path_c != NULL:
        return get_string(path_c).rstrip("header.dxf")
    return None
```